app/helpers.php

```diff
@@ -1,6 +1,34 @@
 <?php
 
 /**
+ * Build a CSV from the supplied data.
+ *
+ * @param  array  $data
+ * @return \League\Csv $csv
+ */
+function buildCSV($data)
```

Comment:
I also just realized that this was just moved up a few lines, so not at all necessary to change in this PR. But maybe worth a future little refactor! :)